bigquery/google/cloud/bigquery/table.py

```diff
@@ -279,6 +279,53 @@ def to_api_repr(self):
             "tableId": self._table_id,
         }
 
+    def to_bqstorage(self, selected_fields=None):
+        """Construct a BigQuery Storage API representation of this table.
+
+        Args:
+            selected_fields (Sequence[ \
+                google.cloud.bigquery.schema.SchemaField, \
+            ]):
+                Optional. A subset of columns to select from this table.
+
+        Returns:
+            Tuple[ \
+                google.cloud.bigquery_storage_v1beta1.types.TableReference, \
+                google.cloud.bigquery_storage_v1beta1.types.TableModifiers, \
+                google.cloud.bigquery_storage_v1beta1.types.TableReadOptions, \
+            ]:
+                A reference to this table in the BigQuery Storage API.
+        """
+        from google.cloud import bigquery_storage_v1beta1
+
+        table_ref = bigquery_storage_v1beta1.types.TableReference()
+        table_ref.project_id = self._project
+        table_ref.dataset_id = self._dataset_id
+
+        modifiers = bigquery_storage_v1beta1.types.TableModifiers()
+        read_options = bigquery_storage_v1beta1.types.TableReadOptions()
+        table_id = self._table_id
+        partition = None
+
+        if "@" in table_id:
+            table_id, snapshot_time = table_id.split("@")
+            snapshot_time = int(snapshot_time)
+            modifiers.snapshot_time.FromMilliseconds(snapshot_time)
+
+        if "$" in table_id:
```

Comment:
Not sure we can assume the $<suffix> is a time partitioned table, with range partitioning on the doorstep.

Reply:
Would someone direct a load job at a specific range partition?